src/citra_qt/main.cpp

```diff
@@ -1985,7 +2041,14 @@ void GMainWindow::OnCaptureScreenshot() {
     const QString timestamp =
         QDateTime::currentDateTime().toString(QStringLiteral("dd.MM.yy_hh.mm.ss.z"));
     path.append(QStringLiteral("/%1_%2.png").arg(filename, timestamp));
-    render_window->CaptureScreenshot(UISettings::values.screenshot_resolution_factor, path);
+
+    auto* const screenshot_window = [this] {
+        if (Settings::values.layout_option != Settings::LayoutOption::SeparateWindows) {
+            return render_window;
+        }
+        return secondary_window->HasFocus() ? secondary_window : render_window;
+    }();
```

Comment:
```suggestion
    GRenderWindow* const screenshot_window = render_window;
    if (Settings::values.layout_option == Settings::LayoutOption::SeparateWindows &&
        secondary_window->HasFocus()) {
        screenshot_window  = secondary_window;
    }
```
The lambda is a bit confusing here and can be simplified